pages.fr/common/du.md

```diff
@@ -0,0 +1,32 @@
+# du
+
+> Utilisation de disque : estime et résume l'utilisation de l'espace occupé par les fichiers et les répertoires.
+> Plus d'informations : <https://www.gnu.org/software/coreutils/du>.
+
+- Liste les tailles d'un répertoire et de ses sous-répertoires, dans l'unité donnée (O/KiO/MiO) :
```

Comment:
```suggestion
- Liste les tailles d'un répertoire et de ses sous-répertoires, dans l'unité donnée (o/ko/Mo) :
```
See https://fr.wiktionary.org/wiki/mégaoctet.

Reply:
On English page, it's B/KiB/MiB (power of 2) not B/KB/MB (power of 10), so it's correct to keep o/Kio/Mio and not o/Ko/Mo.